libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/DirectionsSession.kt

```diff
@@ -3,23 +3,17 @@ package com.mapbox.navigation.core.directions.session
 import com.mapbox.api.directions.v5.models.RouteOptions
 import com.mapbox.navigation.base.route.NavigationRoute
 import com.mapbox.navigation.base.route.NavigationRouterCallback
-import com.mapbox.navigation.base.route.Router
 import com.mapbox.navigation.core.SetRoutes
 
 internal interface DirectionsSession : RouteRefresh {
 
-    /**
-     * Routes that were fetched from [Router] or set manually.
-     * On [routes] change notify registered [RoutesObserver]
-     *
-     * @see [registerRoutesObserver]
-     */
-    val routes: List<NavigationRoute>
```

Comment:
maybe it makes sense to leave `routes` so that you don't need to extract them from the result like here https://github.com/mapbox/mapbox-navigation-android/pull/6745/files#diff-fabf10fcd842f15a333f539689309f950d861096e5f313dbe2b61e771b69a8d6L883 ?

Reply:
Well, maybe, idk, my thinking was that now we have `routesUpdatedResult` that is nullable and it makes us think in places of usage what we want to do it routes have not been set at all yet. While with `routes` we just fall back to the default empty list which is indistinguishable from cleaned routes.
No strong opinion here but that is the reasoning why I did it the way I did it.

Reply:
I think it makes sense to leave `routes` together with `routesUpdatedResult` to avoid long chains in old code like `directionsSession.routesUpdatedResult.routesOrEmpty()`. If somebody needs just routes or empty list they use `routes` if it important to distinguish cleanup from not being set, they use `routesUpdateResult`